src/shared/components/ui/dual-range-slider.tsx

```
@@ -0,0 +1,45 @@
+'use client';
+
+import * as React from 'react';
+import * as SliderPrimitive from '@radix-ui/react-slider';
+
+import { cn } from '@/shared/libs/utils/cn';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)